qt/aqt/addcards.py

```diff
@@ -265,10 +265,6 @@ def _note_can_be_added(self, note: Note) -> bool:
         return True
 
     def keyPressEvent(self, evt: QKeyEvent) -> None:
-        "Show answer on RET or register answer."
-        if evt.key() in (Qt.Key_Enter, Qt.Key_Return) and self.editor.tags.hasFocus():
-            evt.accept()
-            return
```

Comment:
Probably better to remove the whole function, as it's not doing anything anymore.